package.json

```diff
@@ -306,7 +306,7 @@
     "@valist/sdk": "^2.9.1",
     "@vitejs/plugin-react": "^2.2.0",
     "cross-env": "^7.0.3",
-    "electron": "^27.0.0-beta.8",
+    "electron": "^27.0.2",
```

Comment:
this works now? no linux white screen? and no metamask issues?

Reply:
It is the same release so should be fine. The white screen is fixed, we can do more testing with the mm extension as well.